vcstool/commands/import_.py

```diff
@@ -36,12 +41,28 @@ def __init__(
         self.shallow = shallow
 
 
+def file_or_url_type(x):
+    if x.startswith(('http://', 'https://')):
+        try:
+            # use another user agent to avoid getting a 403
```

Comment:
Can you please elaborate when you expect to get a 403 without the `User-Agent` header.

Reply:
done in b4daa32222d568748a6f090127338b1543f71dd9.

As an example, I get a 403 if I use this URL without that non-default user agent: https://gitlab.com/micro-ROS/ros_tracing/ros2_tracing/-/raw/master/instrumented.repos

Reply:
Interesting, I didn't know that about GitLab. Can you update the `User-Agent` to instead of pretending to be `wget` state the actual tool. From briefly reading the docs `vcstool/<version>` sounds like an appropriate user agent string (and it seems to work).

Reply:
done in a5d4092eb667ab28d021246b72381a6cdedfa6d9

I wasn't expecting it to accept a "random"/unknown user agent.